### PR TITLE
New cg-grid for the updated grid sizing

### PR DESCRIPTION
**Pull request by adborden**
Allow us to migrate to a new grid on opt-in bases. USWDS grid is still
available.

@msecret would love to pair with you on this to go over some questions that came up.

### CODE REVIEW

**Comment by adborden** [comment]
@msecret I think this is ready for review, unless there's something specific to do to add it to the styleguide?

I tested this against cg-docs and cg-landing and they are still using the `usa-grid` as expected.

File: src/css/components/grid.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.cg-grid {
```
Comment (msecret) [inline src/css/components/grid.scss:1]: Now that I think about it, we don't use `cg` prefix for anything else. Could it just be `grid`? Or do you think it's a better interface to have the `cg` prefix?

Comment (adborden) [reply src/css/components/grid.scss:1]: Hmm, good point. I'm generally not a fan of prefixes except for frameworks meant to be built on top of. Although cg-style is a framework, it's not meant to be built on, it just contains a bunch of shared stuff. I'll remove the prefix.

**Comment by adborden** [comment]
Also, I noticed on small screens the disclaimer has no padding. There's padding provided by `.content` for the main body grid, but I'm going to get rid of that, I think the grid should handle that kind of thing.

**Review by msecret — APPROVED** [review]

**Pull request merged**